src/main/java/eu/darkbot/utils/Point.java

```diff
@@ -0,0 +1,28 @@
+package eu.darkbot.utils;
+
+import eu.darkbot.api.objects.Locatable;
+
+// in future Location extends Point?
```

Comment:
If location and point are both gonna be double, why would you have 2?

Point is integer based and location is double based, i think that's the intended behaviour